Tests/HederaE2ETests/Contract/Contract.swift

```diff
@@ -65,7 +65,7 @@ extension ContractHelpers {
         let receipt = try await ContractCreateTransaction(
             bytecodeFileId: file.fileId,
             adminKey: operatorAdminKey ? .single(testEnv.operator.privateKey.publicKey) : nil,
-            gas: 100000,
+            gas: 200000,
```

Comment:
Test got more expensive? (if so that's fine, I just want to make sure I understand)

Reply:
Indeed, it has.